README.md

```diff
@@ -63,6 +63,7 @@ alternatives tend to be poorly-received**.
 * Submit a pull request. As a pull request the RFC will receive design
 feedback from the larger community, and the author should be prepared
 to revise it in response.
+* Find a champion on the ember-cli core team. Without a champion, the RFC will not progress.
```

Comment:
What's the motivation behind this?

Also this requires to have a list/link to who are the current people on `ember-cli` core team. Someone relatively new to Ember world won't know who all are on the core team and who he should contact for which area.